### PR TITLE
notes on review of contributions

### PR DESCRIPTION
**Pull request by reneehlozek**
The long overdue PR which includes the review procedure in the READMe for contributions.
 @yymao and @rmandelb please take a look!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,6 +22,21 @@ In the [tutorials directory](https://github.com/LSSTDESC/DC2-analysis/tree/maste
 
 New scripts and notebooks can be added in the [contributed directory](https://github.com/LSSTDESC/DC2-analysis/tree/master/contributed).  The [README](https://github.com/LSSTDESC/DC2-analysis/tree/master/contributed/README.md) has a full description of how to do so.
 
+## Notebook review
+
+There are two reasons why one might add a notebook to the repository:
+
+1) Adding a enerally applicable notebook that you think should be added to the master set of notebooks for DC2 analysis (we imagine the DC2 team will be adding most of these). These will be reviewed by another DC2 analysis team member so the fix/contribution can be added to the tutorial set.
```
Comment (yymao) [inline README.md]:
```suggestion
1) Adding a generally applicable notebook that you think should be added to the master set of notebooks for DC2 analysis (such as a tutorial or a validation notebook; we imagine the DC2 team will be adding most of these). These will be reviewed by another DC2 team member.
```

File: README.md [hunk 2]
```diff
@@ -22,6 +22,21 @@ In the [tutorials directory](https://github.com/LSSTDESC/DC2-analysis/tree/maste
...
+2) You have a notebook for your particular science case that you'd like to add (it could combine some tasks from the existing notebooks, or do something totally different etc.) For these notebooks, we will engage the analysis teams for input.
```
Comment (yymao) [inline README.md]:
```suggestion
2) You have a notebook for your particular science case that you'd like to add (usually under `contributed`; the notebook could combine some tasks from the existing notebooks, or do something totally different etc). For these notebooks, we will engage the analysis teams for input.
```

File: README.md [hunk 3]
```diff
@@ -22,6 +22,21 @@ In the [tutorials directory](https://github.com/LSSTDESC/DC2-analysis/tree/maste
...
+
+Some suggestions for adding new notebooks (e.g. contribution type 2 above). The code should:
+
+a) run without issues, and
```
Comment (rmandelb) [inline README.md]:
```suggestion
a) run without issues; 
```

File: README.md [hunk 4]
```diff
@@ -22,6 +22,21 @@ In the [tutorials directory](https://github.com/LSSTDESC/DC2-analysis/tree/maste
...
+
+There are two reasons why one might add a notebook to the repository:
+
+1) Adding a enerally applicable notebook that you think should be added to the master set of notebooks for DC2 analysis (we imagine the DC2 team will be adding most of these). These will be reviewed by another DC2 analysis team member so the fix/contribution can be added to the tutorial set.
+2) You have a notebook for your particular science case that you'd like to add (it could combine some tasks from the existing notebooks, or do something totally different etc.) For these notebooks, we will engage the analysis teams for input.
+
+Some suggestions for adding new notebooks (e.g. contribution type 2 above). The code should:
+
+a) run without issues, and 
+b) have some texts on what the notebook is trying to do
```
Comment (rmandelb) [inline README.md]:
```suggestion
b) have some explanatory text about what the notebook is trying to do; 
```

File: README.md [hunk 5]
```diff
@@ -22,6 +22,21 @@ In the [tutorials directory](https://github.com/LSSTDESC/DC2-analysis/tree/maste
...
+There are two reasons why one might add a notebook to the repository:
+
+1) Adding a enerally applicable notebook that you think should be added to the master set of notebooks for DC2 analysis (we imagine the DC2 team will be adding most of these). These will be reviewed by another DC2 analysis team member so the fix/contribution can be added to the tutorial set.
+2) You have a notebook for your particular science case that you'd like to add (it could combine some tasks from the existing notebooks, or do something totally different etc.) For these notebooks, we will engage the analysis teams for input.
+
+Some suggestions for adding new notebooks (e.g. contribution type 2 above). The code should:
+
+a) run without issues, and 
+b) have some texts on what the notebook is trying to do 
+c) use recommended packages/instructions whenever possible (e.g., not skipping API to access underlying files) and if not, clearly document their atypical/unrecommended usage
```
Comment (rmandelb) [inline README.md]:
```suggestion
c) use recommended packages/instructions whenever possible (e.g., not skipping API to access underlying files) and if not, clearly document their atypical/unrecommended usage; and
```

File: README.md [hunk 6]
```diff
@@ -22,6 +22,21 @@ In the [tutorials directory](https://github.com/LSSTDESC/DC2-analysis/tree/maste
...
+d) reasonable coding style/practice.
```
Comment (rmandelb) [inline README.md]:
```suggestion
d) follow reasonable coding style/practices (for some examples, see the existing tutorial notebooks; or see the [DESC Coding Guidelines](https://docs.google.com/document/d/1v54bVQI2NejK2UqACDnGXj1t6IGFgY3Uc1R7iV2uLpY/edit?usp=sharing)).
```

**Comment by reneehlozek** [comment]
Also @yymao I added the plot directory and readme to this same PR for simplicity!

**Comment by reneehlozek** [comment]
ok @yymao I removed duplicate info!

**Pull request merged**